user/database-setup.md

```diff
@@ -199,6 +199,23 @@ before_install:
 ```
 {: data-file=".travis.yml"}
 
+### Using `pg_config`
+
+If your builds rely on the use of the `pg_config` command, you need to install an additional
+apt package `postgresql-server-dev-X.Y` for the version of PostgreSQL you are using in order
+to obtain meaningful output from the command.
+
+For example:
+
+```yaml
+addons:
+  apt:
+    packages:
+      - postgresql-server-dev-9.4
```

Comment:
Just to clarify, the X.Y / 9.4 needs to match the existing postgre they are using? So either the default, or the one they specify intentionally?

Reply:
Any one that the user intends to use. This assumes:

```yaml
addons:
  postgresql: "9.4"
```

Perhaps I can add that.

Reply:
That is to say:

```yaml
addons:
  postgresql: "9.4"
  apt:
    packages:
      - postgresql-server-dev-9.4
```

Reply:
How about 

> If your builds rely on the `pg_config` command, you need to install an additional
apt package `postgresql-server-dev-X.Y` where X.Y matches the version of PostgreSQL you are using.

Reply:
That sounds great! :+1: